### DISCUSSION TITLE
autodoc: How to order class attributes before methods with `autoclass`

### DISCUSSION THREAD

**Discussion by surya-sigtech** [sphinx-doc/sphinx#12499] [Q&A]
When generating the class docs using `.. autoclass::`
with the flag `autodoc_member_order = 'groupwise'`

The methods are coming above class attributes. Any suggestions on how to order them?

@tk0miya Is this functionality a part of this open issue #5270

**Comment by surya-sigtech** [comment]
@AA-Turner Any suggestions?

**Reply by AA-Turner** [reply]
Please avoid pinging people for user support queries, it is fairly disruptive. Have you asked on StackOverflow or sought help elsewhere? FYI: I have unsubscribed from this thread.

A

**Comment by jxrossel** [comment]
I've posted an answer here: https://stackoverflow.com/a/79492670/2775385
